aiconvers/claude.py

```diff
@@ -13,7 +14,7 @@ class SystemAction(str, Enum):
 
 class Conversation:
 
-    client = Anthropic(
+    client = AsyncAnthropic(
```

Comment:
oh, amazing, I am impressed. Now we are doing Async . It means 
I was just thinking of it. Additionally, it means it could solve one of the questions that chief officers asked for: faster responses.

You are proactive. Good job. You are ahead of my thoughts.